i3df/src/lib.rs

```diff
@@ -136,7 +137,10 @@ pub fn parse_sector_header(mut input: &mut impl BufRead) -> Result<SectorHeader,
     // read attributes
     let attribute_count = input.read_u32::<LittleEndian>()?;
 
-    if attribute_count != 0 && attribute_count != ATTRIBUTE_COUNT {}
+    // TODO dragly 2020-04-22 should this not return an error?
+    if attribute_count != 0 && attribute_count != ATTRIBUTE_COUNT {
+
+    }
```

Comment:
Shouldn't this jsn't be fixed right away?

Reply:
Done